src/styles/calendar.scss

```diff
@@ -0,0 +1,57 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  padding: 10px;
+  width: calc((#{$day-size} * #{$columns}) + (#{$gap} * #{$columns - 1}));
+  background-color: #fff;
+
+  &__day {
+    width: $day-size;
+    height: $day-size;
+    background-color: #eee;
+    display: flex;
+    border: 1px solid #000;
+    box-sizing: border-box;
+    transition:
+      background-color 0.5s,
+      transform 0.5s;
```

Comment:
The 'transition' property is repeated in the '&:hover' selector. It's better to define the transition once in the '&__day' selector to avoid redundancy.